### PR TITLE
create curator ensemble and close curator for short commit timeout

### PR DESCRIPTION
**Pull request by adyach**
# One-line summary
create curator ensemble and close curator for short commit timeout

### CODE REVIEW

File: src/main/java/org/zalando/nakadi/service/subscription/zk/NewZkSubscriptionClient.java [hunk 1]
```diff
@@ -216,4 +217,8 @@ public void transfer(final String sessionId, final Collection<EventTypePartition
         }
     }
 
+    @Override
+    public void close() throws IOException {
+        getCloseableCuratorFramework().close();
+    }
```
Comment (antban) [inline src/main/java/org/zalando/nakadi/service/subscription/zk/NewZkSubscriptionClient.java]: Very strange. Obtain client in AbstractClient, release - in NewZkClient.

Comment (adyach) [reply src/main/java/org/zalando/nakadi/service/subscription/zk/NewZkSubscriptionClient.java]: this is a mistake, I moved to abstract class

**Comment by antban** [comment]
:+1:

**Comment by adyach** [comment]
👍

**Pull request merged**